### DISCUSSION TITLE
use python boto3 module

### DISCUSSION THREAD

**Discussion by shivatechm** [StackStorm/st2#5476] [Q&A]
I want to add boto3 package in my stackstorm python site-package library how can I do that

**Comment by amanda11** [comment]
I think you have asked the same question here: https://github.com/StackStorm/st2/discussions/5478. I've responded on that track - you need to create a requirements.txt in your pack structure, see https://docs.stackstorm.com/reference/packs.html#anatomy-of-a-pack
